src/uniprotkb/components/entry/ComputationallyMappedSequences.tsx

```diff
@@ -40,9 +45,9 @@ const ComputationalyMappedSequences: FC<{ primaryAccession: string }> = ({
   const columns = useMemo(
```

Comment:
```suggestion
  const columns = useMemo<Array<
    label: string;
    name: string;
    render: (data: ProteinEntryLight) => ReactNode;
  >>(
```
Could probably allow you to remove all the `...: ProteinEntryLight) => ...` below